jobs/sportpursuit-python-developers-x2-roles.html

```diff
@@ -0,0 +1,70 @@
+
+If you’re ambitious, talented, and want to shape the future of the business but don’t mind getting your hands dirty this could be just the place for you.
+
+## Summary
+We are looking for a 2 x Python Developers to assist with development on some key areas of our internal technology platform, namely our Python based ERP platform and Business Intelligence / Data Warehouse system.
+
+## Responsibilities
+* Work in a scrum / agile environment to help develop key features to improve business functionality
+* Backend Python development and provide technical advice to development team members
+* Assist with the design and architecture of the existing data warehouse, with the aim to centralise customer, product and order data.
+* Assist with the technical design and architecture of integration between existing e-commerce platform, ERP and external suppliers.
+* Expand the existing website API capability, specifically around product recommendation and personalisation
+* Expand the existing reporting capabilities to support day-to-day operation of the business.
+* Work alongside internal developers and vendors to customise ERP system to suite business needs
+* Work with key suppliers to build integration to stock and product feeds.
+* Enhance the level of site personalisation, by improving both sport and product recommendation logic.
+* Enhance the machine learning capability for automatic categorisation and tagging of products.
+
+## Skills and Experience
+* Graduate in computer science or related discipline
+* 2-5 years of object orientated Python development experience
+* Development experience using open source Python frameworks ideally Django
+* Exposure to Odoo (OpenERP) ERP platform
+* MySQL or PostgreSQL database development expertise
+* Familiarity with Git for version control and agile practices.
+* A team player with strong written and verbal communication skills
+* Detailed-oriented, with excellent problem solving skills
+
+## Desirable skills
+* Experience using Amazon Web Services (AWS)
+* Working knowledge of AWS data processing and warehouse tools like Redshift, Kinesis, Glue, Athena and Data pipelines.
+* Linux systems administration experience
+* PHP development experience
+
+## Benefits:
+* Workplace pension - we're set up with The Peoples Pension to make sure we're contributing to help you save for retirement.
+* Company bonus scheme – We believe that every member of staff should benefit from company success, so we like to reward everyone accordingly.
+* Holiday – 22 days of holiday a year, and we encourage you take it. We’ll also throw in an extra day off for your birthday.
+* Food & Drink – Hobnobs, tea and coffee, plus all the toast and peanut butter you can eat.
+* Socials – A social budget which grows every time we hit budget, with a social team that just loves using it!
+* The Workspace complex – kitted out with showers, a cafe, and a Pure Gym.
```

Comment:
Is gym membership included in the benefits package, or is it just a facility available in the building?